gui/default/syncthing/device/editDeviceModalView.html

```diff
@@ -127,8 +126,7 @@
                       <input name="maxRecvKbps" id="maxRecvKbps" class="form-control" type="number" ng-model="currentDevice.maxRecvKbps" required="" aria-required="true" min="0"/>
                     </div>
                   </div>
-                  <p class="help-block" ng-if="!deviceEditor.maxRecvKbps.$valid && deviceEditor.maxRecvKbps.$dirty" translate>
-                    The rate limit must be a non-negative number (0: no limit) </p>
+                  <p class="help-block" ng-if="!deviceEditor.maxRecvKbps.$valid && deviceEditor.maxRecvKbps.$dirty" translate>The rate limit must be a non-negative number (0: no limit) </p>
```

Comment:
there are spaces before `</p>`- I don't know if this is a problem for translation strings.